docs/zync-crd-reference.md

```diff
@@ -0,0 +1,140 @@
+# Zync Custom Resource Reference
+
+## Simple CR Example
+
+```yaml
+apiVersion: saas.3scale.net/v1alpha1
+kind: Zync
+metadata:
+  name: simple-example
+spec:
+  image:
+    tag: new-feature
+  secret:
+    vaultPath: secret/data/openshift/dev-example-4-3/3scale-zync
+  zync:
+    replicas: 1
+    env:
+      dbWaitSleepSeconds: 60
+    resources:
+      limits:
+        cpu: 1
+        memory: 1G
+```
+
+## Full CR Example
+
+Most of the fields do not need to be specified (can use default values), this is just an example of everything that can be overriden under your own risk:
+
+```yaml
+apiVersion: saas.3scale.net/v1alpha1
+kind: Zync
+metadata:
+  name: full-example
+spec:
+  image:
+    name: quay.io/3scale/zync
+    tag: nightly
+    pullSecretName: quay-pull-secret
+  secret:
+    vaultPath: secret/data/openshift/dev-example-4-3/3scale-zync
+  zync:
+    replicas: 3
+    env:
+      dbWaitSleepSeconds: 10
+      railsEnv: staging
+      railsLogsToStdout: true
+    resources:
+      requests:
+        cpu: 300m
+        memory: 128Mi
```

Comment:
```suggestion
        memory: "128Mi"
```

Reply:
Quoted in https://github.com/3scale/saas-operator/pull/5/commits/28ab27005b5a702dc7fe996ef20c0004e92850ad